rpmio/rpmkeyring.cc

```diff
@@ -179,7 +179,12 @@ rpmPubkey *rpmGetSubkeys(rpmPubkey primarykey, int *count)
     int pgpsubkeysCount = 0;
     int i;
 
-    if (primarykey && !pgpPrtParamsSubkeys(primarykey->pkt.data(), primarykey->pkt.size(),
+    if (!primarykey)
+	return NULL;
```

Comment:
*count was previously unconditionally set (ie even if key was NULL), so that behavior should be kept, somebody might be relying on it.